cpp/cmake_modules/DefineOptions.cmake

```diff
@@ -346,7 +346,7 @@ takes precedence over ccache if a storage backend is configured" ON)
                 ARROW_WITH_UTF8PROC)
 
   define_option(ARROW_GCS
-                "Build Arrow with GCS support (requires the GCloud SDK for C++)"
+                "Build Arrow with GCS support (requires the Google Cloud Platform C++ Client Libraries)"
```

Comment:
Could you fix this failure?

https://github.com/apache/arrow/actions/runs/14430422114/job/40464526899?pr=46101#step:7:423

```text
CMake Error at cmake_modules/DefineOptions.cmake:27 (message):
  description for ARROW_GCS contained a

          line 86 characters long!
          (max is 80). Split it into more lines with semicolons
Call Stack (most recent call first):
  cmake_modules/DefineOptions.cmake:47 (check_description_length)
  cmake_modules/DefineOptions.cmake:348 (define_option)
  CMakeLists.txt:224 (include)
```

Reply:
Whoops - should be fixed now